src/utils/GovernorDiamondInit.sol

```diff
@@ -73,6 +73,8 @@ contract GovernorDiamondInit {
         config.proposalThresholdToken = proposalThresholdToken;
         // by default, we only allow the default proposal token to be used for proposals
         config.proposalTokens[proposalToken] = true;
+        // by default, we only allow the simple counting strategy to be used for proposals
+        config.countingStrategies[0x6c4b0e9f] = true;
```

Comment:
Where does this `0x6c4b0e9f` come from?

Reply:
> Where does this `0x6c4b0e9f` come from?

```solidity
bytes4(keccak256("simpleWeight(uint256)"))
```

Reply:
It's also used a bit further up in the file and could be swapped out for either that `bytes4` incantation or `GovernorCoreFacet.simpleWeight.selector`.

Reply:
Thank you for the explaination!